src/main/js/components/dropdowns/jumplists.js

```diff
@@ -1,28 +1,30 @@
-import Path from "@/util/path";
-import behaviorShim from "@/util/behavior-shim";
 import Utils from "@/components/dropdowns/utils";
+import behaviorShim from "@/util/behavior-shim";
+import { createElementFromHtml } from "@/util/dom";
+import Path from "@/util/path";
 
 function init() {
   generateJumplistAccessors();
   generateDropdowns();
 }
+function generateDropdownChevron(element) {
```

Comment:
These elements are silent when I try navigate in voiceover:

<img width="461" height="540" alt="Image" src="https://github.com/user-attachments/assets/79e89edf-ce52-4296-87c2-31bec672e755" />

Reply:
Its still silent when navigating the dropdown menu's

Reply:
that code is not used at all! it is a refactor of https://github.com/jenkinsci/jenkins/pull/10803/files#diff-d6982fb188641323a21b35ec8d4aa1c8f31ba1b74506c2d206cea5b59ed7eddcL15 just pulled up to a another function!